data-exporter/src/main/java/org/reactome/release/dataexport/UniProtReactomeEntry.java

```diff
@@ -127,104 +160,140 @@ private UniProtReactomeEntry(long dbId, String accession, String displayName) {
+		uniprotReactomeEntryToReactomeEventCache.put(graphDBSession, uniprotReactomeEntryToReactomeEvent);
 
 		logger.info("Finished computing UniProt to Reactome events");
 
-		return uniprotAccessionToReactomeEvent;
+		return uniprotReactomeEntryToReactomeEvent;
 	}
 
 	/**
-	 * Retrieves, from the graph database, a Map of UniProt accession to the set of identifiers for Reaction Like Events
-	 * in which each UniProt accession participates
+	 * Retrieves, from the graph database, a Map of UniProtReactomeEntry objects to the set of identifiers for
+	 * Reaction Like Events in which each UniProt accession participates
 	 * @param graphDBSession Neo4J Driver Session object for querying the graph database
-	 * @return Map of UniProt accession to set of database identifiers for Reaction Like Events in Reactome
+	 * @return Map of UniProtReactomeEntry objects to set of database identifiers for Reaction Like Events in Reactome
 	 */
-	private static Map<String, Set<Long>> fetchUniProtAccessionToRLEId(Session graphDBSession) {
-		if (uniprotAccessionToReactionLikeEventIdCache.containsKey(graphDBSession)) {
-			return uniprotAccessionToReactionLikeEventIdCache.get(graphDBSession);
+	private static Map<UniProtReactomeEntry, Set<Long>> fetchUniProtReactomeEntryToRLEId(Session graphDBSession) {
+		if (uniprotReactomeEntryToReactionLikeEventIdCache.containsKey(graphDBSession)) {
+			return uniprotReactomeEntryToReactionLikeEventIdCache.get(graphDBSession);
 		}
 
 		logger.info("Computing UniProt to RLE id");
 
 		StatementResult statementResult = graphDBSession.run(
 			String.join(System.lineSeparator(),
 				"MATCH (rgp:ReferenceGeneProduct)<-[:referenceEntity|:referenceSequence|:hasModifiedResidue]-" +
-				"(ewas:EntityWithAccessionedSequence)<-[:hasComponent|:hasMember|:hasCandidate|:repeatedUnit" +
-				"|:input|:output|:catalystActivity|:physicalEntity*]-(rle:ReactionLikeEvent)",
-				"RETURN DISTINCT rgp.identifier, rle.dbId"
+					"(ewas:EntityWithAccessionedSequence)<-[:hasComponent|hasMember|hasCandidate|repeatedUnit" +
+					"|input|output|catalystActivity|physicalEntity*]-(rle:ReactionLikeEvent)",
+				"MATCH (rgp)-[:referenceDatabase]->(rd:ReferenceDatabase)",
+				"WHERE rd.displayName = 'UniProt'",
+				"RETURN DISTINCT rgp.dbId, coalesce(rgp.variantIdentifier, rgp.identifier) as rgp_accession, "
+					+ "rgp.displayName, rle.dbId",
+				"ORDER BY rgp_accession"
 			)
 		);
 
-		Map<String, Set<Long>> uniprotAccessionToReactionLikeEventId = new HashMap<>();
+		Map<UniProtReactomeEntry, Set<Long>> uniprotReactomeEntryToReactionLikeEventId = new HashMap<>();
 		while (statementResult.hasNext()) {
 			Record record = statementResult.next();
 
-			String uniprotAccession = record.get("rgp.identifier").asString();
+			long uniprotDbId = record.get("rgp.dbId").asLong();
+			String uniprotAccession = record.get("rgp_accession").asString();
+			String uniprotDisplayName = record.get("rgp.displayName").asString();
```

Comment:
I've seen these strings ("rgp.displayName") appear in multiple places, and I also noticed for this PR that they were changed from an older format to a newer one. Since they seem to be mutable and also used in multiple places, should they be considered for a Constants class?

Reply:
I _think_ the change from "rgp." to "rgp_" is because of changes to queries, such as this `coalesce(rgp.variantIdentifier, rgp.identifier) as rgp_accession`. Here, `AS` is used to alias a more complex expression to a new name, `rgp_accession` in this case. It should be noted that `rgp_accession` is NOT a name found in any structure within the database (at least I don't think it is). 

I suppose if the same aliases are repeated throughout several queries, creating a set of String constants _might_ be useful. But not necessary.

Reply:
Yeah, I was thinking of consolidating repeated parts of the queries through the PR.  I'll look into it.

Reply:
tbh, I wouldn't worry too much about that.